xml/System/Half.xml

```diff
@@ -784,7 +784,7 @@
       <Docs>
         <param name="s">A string that contains a number to convert.</param>
         <summary>Converts the string representation of a number to its half-precision floating-point number equivalent.</summary>
-        <returns>A half-precision floating-point number equivalent to the numeric value or symbol specified in <paramref name="s"/>.</returns>
+        <returns>A half-precision floating-point number equivalent to the numeric value or symbol specified in <paramref name="s" />.</returns>
```

Comment:
Is adding the missing space a new feature of mdoc 5.8 @joelmartinez ?

Reply:
This is just the built in XML APIs formatting the XML ... hasn't changed in years. But it's a common question from authors who don't expect changes in their written content :)